### PR TITLE
refactoring axios creation and added sentry

### PR DESCRIPTION
**Pull request by saravanpa-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/4648

*Description of changes:*

Added sentry logging.
Refactored axios creation to make it easier.

*Checklist:*
I confirm that below checklist items are addressed in this pull request; (check the boxes applicable)
- [ ] No lint errors
- [ ] No test case failures and proper coverage for all modules/classes
- [ ] Updated the deployment configs for new environment variable(s)
- [ ] Updtaed the postman collection in entity repository (https://github.com/bcgov/entity/tree/master/api-e2e/postman) for e2e tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/util/http-util.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import ConfigHelper from 'sbc-common-components/src/util/config-helper'
```
Comment (nitheesh-aot) [inline auth-web/src/util/http-util.ts:1]: we can use the ConfigHelper from our app itself

File: auth-web/src/util/http-util.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import { SessionStorageKeys } from 'sbc-common-components/src/util/constants'
```
Comment (nitheesh-aot) [inline auth-web/src/util/http-util.ts:2]: same as the above comment

File: auth-web/src/util/http-util.ts (hunk 1 above)
Comment (saravanpa-aot) [reply auth-web/src/util/http-util.ts:1]: haaa..stupid IDE suggested i

**Review by nitheesh-aot — APPROVED** [review]

**Review by sumesh-aot — APPROVED** [review]

**Comment by mengdong19** [comment]
There's build error.

**Pull request merged**